src/server/pkg/storage/fileset/priority_queue.go

```diff
@@ -7,6 +7,7 @@ import (
 type stream interface {
 	next() error
 	key() string
+	streamPriority() int
```

Comment:
Isn't this implied by the ordering of `ss` in `newPriorityQueue` can this change after the fact?